src/calc.py

```diff
@@ -18,3 +18,6 @@ def sub(a, b):
     ```
     """
     return a - b
+
+def mult(a,b):
+    return a * b
```

Comment:
```suggestion
def mult(a,b):
    return a * b
```

Can we call this `x` and `y`?